proxy/http/HttpSM.h

```diff
@@ -101,20 +92,6 @@ enum BackgroundFill_t {
 
 extern ink_mutex debug_sm_list_mutex;
 
-struct HttpVCTableEntry {
-  VConnection *vc;
-  MIOBuffer *read_buffer;
-  MIOBuffer *write_buffer;
-  VIO *read_vio;
-  VIO *write_vio;
-  HttpSMHandler vc_read_handler;
-  HttpSMHandler vc_write_handler;
-  HttpVC_t vc_type;
-  HttpSM *sm;
-  bool eos;
-  bool in_tunnel;
-};
-
 struct HttpVCTable {
```

Comment:
Having `HttpVCTableEntry` and `HttpVCTable` at different places doesn't look like a right thing.

Reply:
Good catch. They should probably go in the same header.